### DISCUSSION TITLE
Corrupted xournal file (.xopp)

### DISCUSSION THREAD

**Discussion by Marcosffn** [xournalpp/xournalpp#3337] [Q&A] [answered]
Hi, my Xournal crashed in the middle of the saving process of one of my xournal file, then I got this error. Luckily, I had exported to PDF before that happen. But I ask myself if it's possible to retrieve some information of the corrupted file, because I don't want to have to redo all of my annotations.
Or maybe there is a autosave copy in somewhere (I already searched in the hidden files of the directory).

![image](https://user-images.githubusercontent.com/67158009/130321386-0ceac745-ca9e-4c18-8389-1b866b8b370d.png)

**Comment by personalizedrefrigerator** [comment, marked as answer]
Please see https://github.com/xournalpp/xournalpp/discussions/3056 for fixing/recovering information from corrupted files.

Additionally, check `~/.config/xournalpp/autosave` for autosaves.

**Reply by Marcosffn** [reply]
Thank you, I founded in that directory an autosave that solved my problem.
